nflows/transforms/nonlinearities.py

```diff
@@ -447,3 +447,53 @@ def forward(self, inputs, context=None):
 
     def inverse(self, inputs, context=None):
         return self._spline(inputs, inverse=True)
+
+class HiLU(Transform):
+
+    def __init__(self, param_shape, init_log_range=0.0):
```

Comment:
Other nonlinearities call this parameter `shape`. Might not be the best name, but I propose sticking to that for consistency.

Reply:
Makes sense.